lib/ui/mail_widget.dart

```diff
@@ -56,6 +60,16 @@ class MailWidgetState extends State<MailWidget> {
       buildLinks();
     }
     stt.setCurrentPage("mail", this);
+    locator<AnalyticsService>().logScreens(name: "signIn");
```

Comment:
is this one supposed to have a different name than signIn? Also, is the commented code below for the future?

Reply:
I fixed that file.  I'll remove the commented code once we have a little more test data.  I also addressed conflicts that were present when the TTS was pulled from the system since it was still identified in this code.